src/renderer/components/AuthModal.ts

```diff
@@ -0,0 +1,92 @@
+import { Authentication } from "@xmcl/user";
+import authModalTemplate from "../templates/AuthModal.pug";
+import { AuthenticationController } from "../controllers/AuthenticationController"; // TODO: temp, remove dep
+import AuthStore from "../store/AuthStore";
+
+export class AuthModal extends HTMLDivElement {
+	public constructor() {
+		super();
+	}
+	public connectedCallback(): void { }
+	public render(message?: string): void {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['message' is defined but never used.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5164335)